docker-compose.yml

```diff
@@ -88,8 +88,23 @@ services:
         tag: "desec-slave/replicator"
     restart: unless-stopped
 
+  prometheus:
+    image: prom/prometheus:latest
```

Comment:
fix major version?

Reply:
The problem is that they don't have an image tag for the major version :-( So if we pin anything, it would be like `v2.15.1`, or we pin nothing. - They release new versions around every 2 weeks, so I figured it would be better to pin nothing.